mantis-control-plane/mantis-control-plane-core/build.gradle

```diff
@@ -32,17 +32,28 @@ dependencies {
     api project(":mantis-common")
 
     api "org.skife.config:config-magic:$configMagicVersion"
+    api 'com.google.code.findbugs:jsr305:3.0.1' // For Nonnull annotation
```

Comment:
does it need to be api level?

Reply:
Based on the code, I think it has to be at the API level.